docs/snippets/route-status-updates.md

```diff
@@ -0,0 +1,90 @@
+## Route status updates
+
+During a roll-out, the system updates the Route and Knative Service status. Both the `traffic` and `conditions` status parameters are affected.
```

Comment:
During a rollout, the system updates the Route and Knative Service status conditions. Both the `traffic` and `conditions` status parameters are affected.